### PR TITLE
docs: Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Unviray**
# Description

The corrections focus on enhancing the `CONTRIBUTING.md` instructions related to Firebase Emulators setup, obtaining the `GOOGLE_CONFIG_BASE64`, and configuring environment variables.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by rhahao — CHANGES_REQUESTED** [review]

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+USER_PARSER_API_KEY=
+GMAIL_ADDRESS=
+GMAIL_APP_PASSWORD=
+GMAIL_SENDER_NAME=
+JW_CDN=https://app.jw-cdn.org/apis/pub-media/GETPUBMEDIALINKS?
+JW_FINDER=https://www.jw.org/finder?
+APP_COUNTRY_API=https://collect-api.sws2apps.com/countries?
+APP_CONGREGATION_API=https://collect-api.sws2apps.com/congregations?
+FIREBASE_APP_NAME=
+FIRESTORE_EMULATOR_HOST=127.0.0.1:8080
```
[the remark this answers is not in the document]
Comment (Unviray) [reply .env.example:10]: Just to double-check, is it intended to be `FIREBASE_STORAGE_EMULATOR_HOST` specifically, and not `STORAGE_EMULATOR_HOST`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -60,6 +71,7 @@ Environment variables are required in order to locally run this API. Please find
 - APP_CONGREGATION_API: set the value to be `https://collect-api.sws2apps.com/congregations?`
 - FIREBASE_APP_NAME: your Firebase Project ID defined in the Firebase Console.
 - FIRESTORE_EMULATOR_HOST: the `Host:Port` for Firestore Emulator.
```
[the remark this answers is not in the document]
Comment (Unviray) [reply CONTRIBUTING.md:73]: Just to double-check, is it intended to be `FIREBASE_STORAGE_EMULATOR_HOST` specifically, and not `STORAGE_EMULATOR_HOST`?

File: .env.example (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhahao) [reply .env.example:10]: Unfortunately, we have to prefix it with `FIREBASE_` per Firebase Docs: https://firebase.google.com/docs/emulator-suite/connect_storage#admin_sdks 😁

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rhahao) [reply CONTRIBUTING.md:73]: Unfortunately, we have to prefix it with `FIREBASE_` per Firebase Docs: https://firebase.google.com/docs/emulator-suite/connect_storage#admin_sdks 😁

**Review by rhahao — APPROVED** [review]

**Pull request merged**